moveit_ros/hybrid_planning/global_planner/global_planner_component/src/global_planner_component.cpp

```diff
@@ -65,7 +65,7 @@ GlobalPlannerComponent::GlobalPlannerComponent(const rclcpp::NodeOptions& option
     }
     else
     {
-      initialized_ = this->init();
+      initialized_ = this->initialize();
```

Comment:
Have you tried the changes to initialize in the constructor instead of in a timer?